src/commands/sentinel-simulate-failure.json

```diff
@@ -11,6 +11,30 @@
             "SENTINEL",
             "ONLY_SENTINEL"
         ],
+        "reply_schema": {
+            "oneOf": [
+                {
+                    "description": "The simulated flag was set.",
+                    "const": "OK"
+                },
+                {
+                    "description": "Supported simulates flags. Returned in case `HELP` was used.",
+                    "type": "array",
+                    "minItems": 2,
```

Comment:
Here, I think we can simply the schema as cluster-help.json and command-help.json